### ISSUE TITLE
Advanced Card Processing gateway not available post-plugin update unless plugin settings are saved once (2081)

### ISSUE THREAD

**Issue by PrimeAuto1** [woocommerce/woocommerce-paypal-payments#1734]
### Describe the Bug

Advanced Card Processing not showing on checkout since latest update.

### To Reproduce

Go to checkout page with advanced card processing enabled.

### Screenshots

https://i.imgur.com/rLw7ppW.png

### Expected Behavior

Credit card payment as an option on checkout.

### Actual Behavior

No advanced card processing/credit card option on checkout.

### Environment

 - WooCommerce Version 8.1.1
 - Plugin Version Version 2.3.0 and Version 2.3.1
 - Any Browser

### Additional Details

<details><summary>System status</summary>

  ```
  <!--If applicable, paste the system status here. Please ensure you redact or remove any identifying information. -->
  ```

</details>

**Comment by InpsydeNiklas** [comment]
Hi @PrimeAuto1, thanks for reporting this issue. Could you please try saving the Advanced Card Processing settings to see if the gateway appears for you?

**Comment by PrimeAuto1** [comment]
After disabling and re-enabling it is working now. We use Curcy Multiple Currency plugin and the only option earlier was Paypal but it is showing the Advanced Card Processing as an option now. Thank you

**Comment by InpsydeNiklas** [comment]
Thank you for the feedback. When switching between current and previous versions, the Advanced Card Processing gateway may not display anymore despite being enabled.
But saving any PayPal Payments settings tab appears to resolve the issue. No need to disable and re-enable Advanced Card Processing as any save action in any tab of the PayPal Payments plugin settings should update the gateway status.

**Comment by InpsydeNiklas** [comment]
FYI this issue of the Advanced Card Processing gateway disappearing after the plugin update has been resolved for good with version 2.4.0 and newer. Thanks again for the feedback.
